app/assets/javascripts/components/story/DatePicker.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+
+export default class StoryDatePicker extends React.Component {
+  constructor (props) {
+    super(props)
+    moment.locale(I18n.locale);
+    const releaseDate = moment(this.props.releaseDate, ["YYYY-MM-DD"])
+    this.state = {
+      startDate: releaseDate.isValid() ? releaseDate : null
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(date) {
+    this.setState({
+      startDate: date
+    }, function(){$('input[name=release_date]').trigger('change')});
```

Comment:
I think it's better to call the listener function directly instead of simulating an event, using jQuery inside React is __bad__.